recipe/meta.yaml

```diff
@@ -12,30 +12,21 @@ source:
   sha256: {{ checksum }}
 
 build:
-  number: 0
+  number: 1
+  # These are present when the new environment is created
+  # so we have to exempt them from the list of initial files
+  # for conda-build to realize they should be included.
   entry_points:
     - conda = conda.cli:main
     - conda-env = conda_env.cli.main:main
   always_include_files:
     - bin/conda                            # [unix]
     - bin/activate                         # [unix]
     - bin/deactivate                       # [unix]
-    - etc/fish/conf.d/conda.fish           # [unix]
     - Scripts/activate.bat                 # [win]
     - Scripts/activate                     # [win]
-    - Scripts/conda.exe                    # [win]
-    - Scripts/conda-script.py              # [win]
-    - Scripts/conda-script.pyc             # [win]
```

Comment:
Conda 4.2.13 won't be *in* the environment.  The executables will be symlinked into the environment.  Which is why we have extra hoops to jump through here to deal with that.  The fish reference isn't needed in `always_include_files`, only the paths that conda symlinks into an environment when the environment is activated.  That complete list, I *think*, is

```
    - bin/conda                 [unix]
    - bin/activate              [unix]
    - bin/deactivate            [unix]
    - Scripts/activate.bat      [win]
    - Scripts/deactivate.bat    [win]
    - Scripts/activate          [win]
    - Scripts/deactivate        [win]
```

This is for both conda 4.2 and conda 4.3.  Conda 4.4 will be a little bit different here.  But again, we'll deal with that later :)